src/libs/ValidationUtils.ts

```diff
@@ -1,19 +1,19 @@
 import {subYears, addYears, startOfDay, endOfMonth, parse, isAfter, isBefore, isValid, isWithinInterval, isSameDay, format} from 'date-fns';
-import _ from 'underscore';
 import {URL_REGEX_WITH_REQUIRED_PROTOCOL} from 'expensify-common/lib/Url';
 import {parsePhoneNumber} from 'awesome-phonenumber';
+import isDate from 'lodash/isDate';
+import isEmpty from 'lodash/isEmpty';
+import isObject from 'lodash/isObject';
 import CONST from '../CONST';
 import * as CardUtils from './CardUtils';
 import * as LoginUtils from './LoginUtils';
+import {Report} from '../types/onyx';
 
 /**
  * Implements the Luhn Algorithm, a checksum formula used to validate credit card
  * numbers.
- *
- * @param {String} val
- * @returns {Boolean}
  */
-function validateCardNumber(val) {
+function validateCardNumber(val: string): boolean {
```

Comment:
Let's rename it. Moreover, across the file they use `value`.
```suggestion
function validateCardNumber(value: string): boolean {
```